doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -121,6 +121,80 @@ If the difference in the values of the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRA
 
 To enable logging of the modem trace bitrate, enable the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_LOG` Kconfig.
 
+.. _modem_trace_backend_uart_nrf9160dk:
+
+Sending traces over UART on the nRF9160DK
+=========================================
+
+When sending modem traces over UART1 on the nRF9160DK, configuration must be added for the UART device in the devicetree.
+This is done by adding the following code snippet to the board devicetree or overlay file.
```

Comment:
Is this `.rst` file inherited from modem library, or is there some other reason why the `west build -S` snippet trick is not mentioned here.

Can we add it here as well? Someone might end up reading this page, without knowledge of the added snippet.